contact.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Contact us - If Else Network</title>
+		<link rel="stylesheet" href="main.css?v=1" />
+		<script>
+			/* Sanity check for local development */
+			if (window.location.href.match("http") && window.location.host.split(':')[0] !== '127.0.0.1' && window.location.host.indexOf('file:///') < 0) {
+				if (!window.location.href.match("https")) {
+					window.location.href = window.location.href.replace(
+						"http",
+						"https"
+					);
+				}
+			}
+		</script>
+
+		<!-- Social media cards -->
+		<meta name="twitter:card" content="summary" />
+		<meta name="twitter:site" content="@ifelsenetwork" />
+		<meta
+			name="twitter:title"
+			content="Contact the  If Else Network staff!"
+		/>
+		<meta
+			name="twitter:description"
+			content="The IfElse Network empowers the IT industry, especially its younger members, with communication and productivity tools."
+		/>
+		<meta
+			name="twitter:image"
+			content="assets/IfElseLogo.png"
+		/>
+		<meta name="twitter:image:alt" content="If Else Logo" />
+
+		<!-- Icons and mobile things -->
+
+		<link
+			rel="apple-touch-icon"
+			sizes="180x180"
+			href="assets/apple-touch-icon.png?v=1"
+		/>
+		<link
+			rel="icon"
+			type="image/png"
+			sizes="32x32"
+			href="assets/favicon-32x32.png?v=1"
+		/>
+		<link
+			rel="icon"
+			type="image/png"
+			sizes="194x194"
+			href="assets/favicon-194x194.png?v=1"
+		/>
+		<link
+			rel="icon"
+			type="image/png"
+			sizes="192x192"
+			href="assets/android-chrome-192x192.png?v=1"
+		/>
+		<link
+			rel="icon"
+			type="image/png"
+			sizes="16x16"
+			href="assets/favicon-16x16.png?v=1"
+		/>
+		<link rel="manifest" href="assets/site.webmanifest?v=1" />
+		<link rel="shortcut icon" href="assets/favicon.ico?v=1" />
+		<meta name="apple-mobile-web-app-title" content="About If Else" />
+		<meta name="application-name" content="If Else" />
+		<meta name="msapplication-TileColor" content="#603cba" />
+		<meta
+			name="msapplication-TileImage"
+			content="assets/mstile-144x144.png?v=1"
+		/>
+		<meta
+			name="msapplication-config"
+			content="assets/browserconfig.xml?v=1"
+		/>
+		<meta name="theme-color" content="#2b2b2b" />
+	</head>
+	<body>
+		<main>
+			<h2>Why not</h2>
+			<!-- When the app is released, this link will point to it instead -->
+			<h1>
+				<a
+					class="title-link"
+					href="https://about.ifelse.network/contact.html"
+					rel="noopener"
+					>contact us?</a
+				>
+			</h1>
+			<h4>I mean, email us.</h4>
+			<footer>
+				<a
+					class="homepage-link"
+					href="mailto:support@ifelse.network"
```

Comment:
Exposing email addresses in plaintext is a good way to get spammed by bots. Just saying.